fedoragsearch-transforms/islandora_transforms/slurp_all_MODS_to_solr.xslt

```diff
@@ -34,77 +34,35 @@
       mods:genre = 'Academic theses'.
     -->
     <xsl:apply-templates mode="utk_ir_MODS" select="$content//mods:mods[1]"/>
-	<xsl:apply-templates mode="utk_mods_etd" select="$content//mods:mods[1]"/>
   </xsl:template>
 
   <!-- utk_ir_MODS mode -->
-  <!-- the following template creates an _ms field for author(s) -->
-  <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Author') or (mods:role/mods:roleTerm='author')]" mode="utk_ir_MODS">
-    <xsl:variable name="given-n" select="mods:namePart[@type='given']"/>
-    <xsl:variable name="family-n" select="mods:namePart[@type='family']"/>
-    <xsl:variable name="t-o-address" select="mods:namePart[@type='termsOfAddress']"/>
-
-    <field name="utk_mods_ir_name_author_ms">
-      <xsl:choose>
-        <xsl:when test="$t-o-address!=''">
-          <xsl:value-of select="concat($given-n, ' ', $family-n, ', ', $t-o-address)"/>
-        </xsl:when>
-        <xsl:otherwise>
-          <xsl:value-of select="concat($given-n, ' ', $family-n)"/>
-        </xsl:otherwise>
-      </xsl:choose>
-    </field>
-  </xsl:template>
 
-  <!-- utk_mods_etd mode -->
-  <!-- utk_mods_etd mode --> 
 
-
-  <!-- JIRA TRAC-875 Define utk_mods_etd_author in Solr -->
+  <!-- JIRA TRAC-875 Define utk_mods_etd_name_author_ms in Solr -->
   <!-- the following template creates an _s field for single etd author -->
   <!-- need to  use single_valued_hashset -->
   <xsl:template match="mods:mods/mods:name[(mods:role/mods:roleTerm='Author') or 
-    (mods:role/mods:roleTerm='author')]" mode="utk_mods_etd">
+    (mods:role/mods:roleTerm='author')]" mode="utk_ir_MODS">
     <xsl:variable name="given-n" select="mods:namePart[@type='given']"/>
     <xsl:variable name="family-n" select="mods:namePart[@type='family']"/>
     <xsl:variable name="t-o-address" select="mods:namePart[@type='termsOfAddress']"/>
 
-     <field name="utk_etd_mods_name_author_s">
+     <field name="utk_mods_name_author_ms">
```

Comment:
@cdeaneGit everything looks great except for this one field: please add '_etd_' to the field name; e.g. `utk_mods_etd_name_author_ms`.

Reply:
Sorry for missing that.  Yes, add those 4 characters, add, commit, and push so we can merge.

Reply:
Done.  utk_mods_etd_name_author_ms